File: AirportSimulatorFrontend/src/app/shared/Request.ts
import { Flight } from './Flight'

export interface Request{

    id?: number;
    flight: Flight;
    total: number;
    type:string;
    time:Date;
    created:Date;
    aproved:boolean;

}

export class Request implements Request{

    id?: number;
    flight: Flight;
    total: number;
    type:string;
    time:Date;
    created:Date;
    aproved:boolean;

}

File: AirportSimulatorBackend/Models/Request.cs
using System;

namespace AirportSimulatorBackend.Models
{
    public class Request
    {
        public int Id { get; set; }
        public Flight Flight { get; set; }
        public string Type { get; set; }
        public DateTime Time { get; set; }
        public bool aproved { get; set; }
        public bool active { get; set; }
        public DateTime Created { get; set; }
    }
}